packages/portal/src/components/page/PageCookiesWidget.vue

```diff
@@ -1,5 +1,7 @@
 <template>
-  <div>
+  <div
+    v-if="cookieConsentRequired"
+  >
```

Comment:
Probably remove this as the widget is also used in the EmbedGateway when cookieConsentRequired can be false

Reply:
Refactored to handle both scenarios.